llama_index/graph_stores/nebulagraph.py

```diff
@@ -16,11 +16,17 @@
 logger = logging.getLogger(__name__)
 
 
-rel_query = Template(
+rel_query_0 = Template(
```

Comment:
nit: can we give these a more descriptive name?

Reply:
Will do so :)